pkg/gui/branches_panel.go

```diff
@@ -76,6 +76,10 @@ func (gui *Gui) deleteBranch(g *gocui.Gui, v *gocui.View, force bool) error {
 	if checkedOutBranch.Name == selectedBranch.Name {
 		return gui.createErrorPanel(g, gui.Tr.SLocalize("CantDeleteCheckOutBranch"))
 	}
+	return gui.deleteNamedBranch(g, v, selectedBranch, force)
+}
+
+func (gui *Gui) deleteNamedBranch(g *gocui.Gui, v *gocui.View, selectedBranch *commands.Branch, force bool) error {
 	title := gui.Tr.SLocalize("DeleteBranch")
 	var messageId string
 	if force {
```

Comment:
Now that we only get the "ForceDeleteBranchMessage" when the user has already tried doing a regular branch delete, we should change the message to give more context. Currently the first prompt comes up and says 
`Are you sure you want to delete the branch my-branch?`
then you hit enter. Then the next message comes up:
`Are you sure you want to force delete the branch my-branch?`
This second message, i.e. `ForceDeleteBranchMessage` should read:
`my-branch is not fully merged. Are you sure you want to delete it?`

This patch will do the trick (it also removes the force delete keybinding because we don't need that anymore):
```
diff --git a/pkg/gui/keybindings.go b/pkg/gui/keybindings.go
index 1073da6..f933468 100644
--- a/pkg/gui/keybindings.go
+++ b/pkg/gui/keybindings.go
@@ -295,12 +295,6 @@ func (gui *Gui) GetKeybindings() []*Binding {
                        Modifier:    gocui.ModNone,
                        Handler:     gui.handleDeleteBranch,
                        Description: gui.Tr.SLocalize("deleteBranch"),
-               }, {
-                       ViewName:    "branches",
-                       Key:         'D',
-                       Modifier:    gocui.ModNone,
-                       Handler:     gui.handleForceDeleteBranch,
-                       Description: gui.Tr.SLocalize("forceDeleteBranch"),
                }, {
                        ViewName:    "branches",
                        Key:         'm',
diff --git a/pkg/i18n/english.go b/pkg/i18n/english.go
index b8be4cf..4682e76 100644
--- a/pkg/i18n/english.go
+++ b/pkg/i18n/english.go
@@ -167,7 +167,7 @@ func addEnglish(i18nObject *i18n.Bundle) error {
                        Other: "Are you sure you want to delete the branch {{.selectedBranchName}}?",
                }, &i18n.Message{
                        ID:    "ForceDeleteBranchMessage",
-                       Other: "Are you sure you want to force delete the branch {{.selectedBranchName}}?",
+                       Other: "{{.selectedBranchName}} is not fully merged. Are you sure you want to delete it?",
                }, &i18n.Message{
                        ID:    "CantMergeBranchIntoItself",
                        Other: "You cannot merge a branch into itself",
```

Reply:
Have applied the patch.